apps/ledger-live-desktop/src/newArch/features/ModularDrawer/hooks/useModularDrawerRemoteData.ts

```diff
@@ -60,7 +60,7 @@ export function useModularDrawerRemoteData({
   });
 
   return {
-    error,
+    error: !!error, // Convert to boolean to avoid exposing error details (HTML from 500 responses)
```

Comment:
Sorry I think I'm missing something - the usage of `useModularDrawerRemoteData` in `ModularDrawerFlowManager` passes error into `AssetSelection` like `error={!!error}` already doesn't it? How does this fix the issue?